packages/core/src/hosted-form/iframe-content/hosted-input-validator.ts

```diff
@@ -42,7 +42,7 @@ export default class HostedInputValidator {
 
             return results;
         } catch (error) {
-            if (error.name !== 'ValidationError') {
+            if (this._isValidErrorType(error) && error.name !== 'ValidationError') {
```

Comment:
is that a mistake?
should it be?
```if (!this._isValidErrorType(error)) {```

Reply:
I believe it still works as expected, but typeguard can be written differently.